### PR TITLE
Add default username for Redis

### PR DESCRIPTION
**Pull request by gzdunek**
Closes https://github.com/gravitational/webapps.e/issues/320 

I was unsure whether I should pass `default` value from the `MenuLogin` picker when it is empty (for redis protocol) or it is better to do have a default value in `connect` in `useDatabases`. IMO the second way is more universal 🤔

### CODE REVIEW

File: packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx [hunk 1]
```diff
@@ -83,7 +86,8 @@ function ConnectButton({
     <Cell align="right">
       <MenuLoginTheme>
         <MenuLogin
-          placeholder="Enter username"
+          {...getMenuLoginOptions(protocol)}
+          width="195px"
```
Comment (ravicious) [inline packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx:90]: As much as I dislike setting a specific width to elements, I can't really think of a better solution here. ;f

File: packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/useDatabases.ts [hunk 2]
```diff
@@ -36,12 +36,20 @@ export function useDatabases() {
...
-      targetUser: dbUser,
+      targetUser: getTargetUser(db.protocol, dbUser),
     });
     documentsService.add(doc);
     documentsService.open(doc.uri);
   }
 
+  function getTargetUser(protocol: string, providedDbUser: string): string {
+    if (protocol === 'redis') {
+      return providedDbUser || 'default';
```
Comment (ravicious) [inline packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/useDatabases.ts:47]: Could you mention in a comment that we're replicating [tsh logic](https://github.com/gravitational/teleport/blob/796e37bdbc1cb6e0a93b07115ffefa0e6922c529/tool/tsh/db.go#L240-L244) here but unlike tsh, in Connect each gateway document assumes that the username is present?

File: packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx [hunk 3]
```diff
@@ -71,10 +72,12 @@ function DatabaseList(props: State) {
 function ConnectButton({
   documentUri,
   dbUri,
+  protocol,
   onConnect,
 }: {
   documentUri: string;
   dbUri: string;
+  protocol: string;
```
Comment (ravicious) [inline packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx:80]: Could you use `GatewayProtocol` as the type here? https://github.com/gravitational/webapps/blob/79547b2d7399bfa611e3fd1e9dd04452a718ad2b/packages/teleterm/src/services/tshd/types.ts#L13-L23

**Review by ravicious — APPROVED** [review]
I'd just add a comment explaining why Redis is getting a special treatment here but other than that it looks fine.

Defining the default username in `useDatabases` seems like a good move to me. 👍

File: packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/useDatabases.ts (hunk 2 above)
Comment (gzdunek) [reply packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/useDatabases.ts:62]: Done

File: packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx (hunk 3 above)
Comment (gzdunek) [reply packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx]: Done

**Pull request merged**